docs/toc.md

```diff
@@ -0,0 +1,23 @@
+# Welcome to Multicast' documentation!
+
+## Contents:
+
+```{toctree}
+:maxdepth: 3
+apidocs/index
+/README.md
+/setup.py
+/LICENSE.md
+:Name: Documentation
+/
+```
+
+## Overview
+
+```{autosummary}
+```
+
+
+## Quickstart:
+
+more filler text
```

Comment:
Good catch. This will need to be changed, but let's actually make a quickstart:

* Read './README.md' and review the planned changes (W.I.P) from #104 and consider what a hypothetical novice python developer would want from a possible quick-start guide. Let's call that "the possible quick-start plan"
* Use "the possible quick-start plan" to generate 4-6 quick-start guides, varying in tone from either a tone that sounds like your talking to a 5-year-old or a tone that sounds like a typical college text-book about computer-science.

Reply:
Let's go with the 2nd option "Friendly and Approachable" ( although I really appreciate the "Formal Academic Tone" too) 😅 

Let's integrate our new quick-start guide into the toc.md in place of the filler text.

Reply:
LGTM! Thus we have the following added to the batch:
```suggestion
## Quickstart:
**Welcome to the Multicast Python Library! Let's get you started quickly.**

**Step 1: Install Python 3**

* Ensure Python 3 is installed on your system.

**Step 2: Clone the Repository**

* Open your terminal and run:
  ```shell
  git clone https://github.com/reactive-firewall/multicast.git
  cd multicast
  ```

**Step 3: Install the Package**

* Run:
  ```shell
  make install
  ```

**Step 4: Sending Messages**

* Send a message using:
  ```shell
  python3 -m multicast SAY --mcast-group 224.1.1.2 --port 59595 --message "Hello, Multicast!"
  ```

**Step 5: Receiving Messages**

* Receive messages by running:
  ```shell
  python3 -m multicast RECV --use-std --port 59595 --join-mcast-groups 224.1.1.2
```

**You're all set! Enjoy using Multicast for your projects.**

Reply:
🙅🏻 OPPS! github goofed on parsing that one. I'll have to do these changes from my local copy.